src/post_setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+sed -i '/^users:/ {N; s/users:.*/users: []/g}' /etc/cloud/cloud.cfg
```

Comment:
Prefer the long form of CLI options:
```suggestion
sed --in-place '/^users:/ {N; s/users:.*/users: []/g}' /etc/cloud/cloud.cfg
```